x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -804,30 +802,23 @@
     "xpack.lens.xyChart.nestUnderRoot": "データセット全体",
     "xpack.lens.shared.overwriteAxisTitle": "軸タイトルを上書き",
     "xpack.lens.xyChart.position.help": "凡例の配置を指定します。",
-    "xpack.lens.xyChart.referenceLine.alertIconLabel": "アラート",
-    "xpack.lens.xyChart.referenceLine.asteriskIconLabel": "アスタリスク",
-    "xpack.lens.xyChart.referenceLine.bellIconLabel": "ベル",
-    "xpack.lens.xyChart.referenceLine.boltIconLabel": "ボルト",
-    "xpack.lens.xyChart.referenceLine.bugIconLabel": "バグ",
-    "xpack.lens.xyChart.referenceLine.commentIconLabel": "コメント",
-    "xpack.lens.xyChart.referenceLine.flagIconLabel": "旗",
-    "xpack.lens.xyChart.referenceLine.noIconLabel": "なし",
-    "xpack.lens.xyChart.referenceLine.tagIconLabel": "タグ",
-    "xpack.lens.xyChart.referenceLineColor.label": "色",
-    "xpack.lens.xyChart.referenceLineFill.above": "上",
```

Comment:
Why did you remove some translations here? Because they are used in a new context now?

Reply:
I am not sure what happened here, I didn't remove it on purpose 🙈  I'll revise, thank you!